Portal/View/UIKit/Renderers/TextFieldRenderer.swift

```diff
@@ -29,30 +30,32 @@ internal struct TextFieldRenderer<MessageType, RouteType: Route>: UIKitRenderer
         textField.removeTarget(.none, action: .none, for: .editingDidBegin)
         textField.removeTarget(.none, action: .none, for: .editingChanged)
         textField.removeTarget(.none, action: .none, for: .editingDidEnd)
+        if properties.shouldReturn {
+            textField.delegate = TextFieldDelegate.shared
+        }
 
-        let mailbox: Mailbox<ActionType> = textField.bindMessageDispatcher { mailbox in
-            properties.onEvents.onEditingBegin |> { _ = textField.dispatch(
-                message: $0,
-                for: .editingDidBegin,
-                with: mailbox)
-            }
-            properties.onEvents.onEditingChanged |> { _ = textField.dispatch(
-                message: $0,
-                for: .editingChanged,
-                with: mailbox)
-            }
-            properties.onEvents.onEditingEnd |> { _ = textField.dispatch(
-                message: $0,
-                for: .editingDidEnd,
-                with: mailbox)
-            }
+        let dispatcher: (Mailbox<ActionType>) -> Void = { mailbox in
```

Comment:
Why are you using a variable to store the closure instead of passing this closure directly to `bindMessageDispatcher`?